testground/benchmark/benchmark/stateless.py

```diff
@@ -307,15 +308,52 @@ def wait_for_peers(home: Path):
         wait_for_port(ECHO_SERVER_PORT, host=host, timeout=2400)
 
 
+def truncate_fractional_seconds(timestamp):
+    (
+        date_time_part,
+        _,
+        _,
+    ) = timestamp.partition("Z")
+    if "." in date_time_part:
+        date_time_part, fractional_part = date_time_part.split(".")
+        fractional_part = fractional_part[:6]
+        return f"{date_time_part}.{fractional_part}Z"
+    return timestamp
+
+
+def calculate_tps(height, num_txs, timestamp):
```

Comment:
we might need to do average for a range of blocks, `block size / block time` for a single block might not be meaningful.